tests/tool_use/utils.py

```diff
@@ -111,7 +111,18 @@ def ensure_system_prompt(messages: List[Dict[str, Any]],
         ],
         "supports_parallel":
         False,
-    }
+    },
+    "pythonic": {
```

Comment:
can you name this one `llama3.2` and create a separate one for Tool ACE once the chat template is ready?

Reply:
Done.